rizinorg/rizin
# meson: use major/minor only in the lib soversion

**Your checklist for this pull request**

**Detailed description**



Before:
```
lrwxrwxrwx. 1 rschiron rschiron       23 May  9 17:09 /home/rschiron/.local/lib64/librz_type.so -> librz_type.so.0.4.0-git                                                                                        
-rwxr-xr-x. 1 rschiron rschiron  1269744 May  9 17:09 /home/rschiron/.local/lib64/librz_type.so.0.4.0-git
```
After:
```
lrwxrwxrwx. 1 root root   17 May 10 10:16 /lib/x86_64-linux-gnu/librz_type.so -> librz_type.so.0.4
lrwxrwxrwx. 1 root root   19 May 10 10:16 /lib/x86_64-linux-gnu/librz_type.so.0.4 -> librz_type.so.0.4.1
-rwxr-xr-x. 1 root root 2.0M May 10 10:16 /lib/x86_64-linux-gnu/librz_type.so.0.4.1
-rwxr-xr-x. 1 root root 2.0M May 10 10:12 /lib/x86_64-linux-gnu/librz_type.so.0.4.1-git
```

**Test plan**



1) install rizin from this PR
2) build a simple program against rz_util
3) run it, it should work
4) uninstall rizin; update the version field of the rizin directory; re-install it.
5) run again the test program and it should work, now calling the newer rz_util lib version

**Closing issues**



Closes #2290
